### PR TITLE
Adding Custom Action Project Documentation

### PR DESCRIPTION
**Pull request by erdomke**
- Adding documentation on changes necessary for Custom Action projects
- Tweaking document formatting

### CODE REVIEW

File: src/Web/Static/documents/documentation/manual/v3/msbuild/daily_builds.html.md [hunk 1]
```diff
@@ -30,4 +30,18 @@ The WixToolPath must be set to point to the location of the WiX tools directory
...
+      &lt;</font><font size="2" color="#A31515">WixToolPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">$(SourceCodeControlRoot)\wix\[[Version]]\</font><font size="2" color="#0000FF">\&lt;/</font><font size="2" color="#A31515">WixToolPath</font><font size="2" color="#0000FF">&gt;
+      &lt;</font><font size="2" color="#A31515">WixTargetsPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">$(WixToolPath)Wix.targets</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">WixTargetsPath</font><font size="2" color="#0000FF">&gt;
+      &lt;</font><font size="2" color="#A31515">WixTasksPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">$(WixToolPath)wixtasks.dll</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">WixTasksPath</font><font size="2" color="#0000FF">&gt;
+&lt;</font><font size="2" color="#A31515">/PropertyGroup</font><font size="2" color="#0000FF">&gt;</font></pre>
+
+Also, be certain to reference the local copy of the `Microsoft.Deployment.WindowsInstaller.dll` as opposed to the version installed in the GAC.
```
Comment (robmen) [inline src/Web/Static/documents/documentation/manual/v3/msbuild/daily_builds.html.md]: Hmm, DTF should not be in the GAC. I don't think this comment makes sense.

Comment (erdomke) [reply src/Web/Static/documents/documentation/manual/v3/msbuild/daily_builds.html.md]: Perhaps I should change the verbiage.  I believe I had initially created the project from a template, and there was a reference to the dll in `C:\Program Files (x86)\WiX Toolset v3.11\SDK`.  I had to modify this to a local reference to get the project to build on the build server.  Would the phrase "... version installed on the development computer" or something similar be better?

Comment (robmen) [reply src/Web/Static/documents/documentation/manual/v3/msbuild/daily_builds.html.md]: Yes, that verbiage would be better. We do not recommend nor condone GAC'ing WiX assemblies. :smile:

IIRC (and I don't use DTF myself), you can use the WIX environment var (like `$(Wix)\sdk\...`) to make the project more portable.

Comment (rseanhall) [reply src/Web/Static/documents/documentation/manual/v3/msbuild/daily_builds.html.md]: Note that this file is autogenerated, the source for wix3 is at https://github.com/wixtoolset/wix3/blob/develop/src/chm/documents/msbuild/daily_builds.html.md.

I think that it should recommend using `$(WixSdkPath)` (or `$(WixToolsPath)`?) as the root for the reference to any WiX dlls.

**Pull request closed without merging**